src/core/connection.c

```diff
@@ -6617,6 +7055,28 @@ QuicConnParamSet(
         return QUIC_STATUS_SUCCESS;
     }
 
+    case QUIC_PARAM_CONN_ADD_LOCAL_ADDRESS: {
+
+        if (BufferLength != sizeof(QUIC_ADDR)) {
```

Comment:
Let's move `QuicAddrIsValid(LocalAddress)` to here, instead of in `QuicConnAddLocalAddress`.